### PR TITLE
Fixed integration tests

### PR DESCRIPTION
**Pull request by iulianpascalau**
- removed advertiser from processing integration tests
- split vm/esdt in multiple packages

TODO: fix TestIssueESDT_FromSCWithNotEnoughGas test

### CODE REVIEW

File: integrationTests/node/heartbeat/heartbeat_test.go [hunk 1]
```diff
@@ -150,6 +141,12 @@ func prepareNodes(
 		_ = nodes[i].Bootstrap()
 	}
 
+	for i := 0; i < len(nodes); i++ {
```
Comment (mihaib79) [inline integrationTests/node/heartbeat/heartbeat_test.go:144]: i < len(nodes) - 1 ?

File: integrationTests/testInitializer.go [hunk 2]
```diff
@@ -1157,52 +1145,97 @@ func CreateNodes(
...
+	ConnectNodes(connectableNodes)
+
 	return nodes
 }
 
+// ConnectNodes will try to connect all provided connectable instances in a full mesh fashion
+func ConnectNodes(nodes []Connectable) {
+	encounteredErrors := make([]error, 0)
+
+	for i := 0; i < len(nodes); i++ {
```
Comment (mihaib79) [inline integrationTests/testInitializer.go:1178]: i < len(nodes) - 1 ?

**Review by mihaib79 — APPROVED** [review]

File: integrationTests/node/heartbeat/heartbeat_test.go (hunk 1 above)
Comment (iulianpascalau) [reply integrationTests/node/heartbeat/heartbeat_test.go]: done, thanks

File: integrationTests/testInitializer.go (hunk 2 above)
Comment (iulianpascalau) [reply integrationTests/testInitializer.go]: done, thanks

**Review by mihaib79 — APPROVED** [review]

**Review by raduchis — APPROVED** [review]

**Review by LucianMincu — APPROVED** [review]

**Pull request merged**